PsevdoTwwiterApp/src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="resources/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <title>PsevdoTwitter</title>
+
+  <script type="text/javascript" src="resources/js/script.js"></script>
+</head>
+
+<body>
+
+  <button class="Log" id="logOut">Without logginig </button>
+  <button class="LogIn" id="logIn">Log in</button>
+  <button class="LogOutforV" id="logOutForV">Log Out</button>
+  <div class="log" id="logPage">
+
+    <div class="limiter">
+      <div class="container-login100">
+        <div class="wrap-login100 p-l-85 p-r-85 p-t-55 p-b-55">
+          <div class="login100-form validate-form flex-sb flex-w">
+            <span class="login100-form-title p-b-32">
+              Account Login
+            </span>
+
+            <span class="txt p-b-11">
+              Username
+            </span>
+            <div class="wrap-input100 validate-input m-b-36">
+              <input id="userInputName" class="input100" type="text" name="username">
+            </div>
+
+            <span class="txt p-b-11">
+              Password
+            </span>
+            <div class="wrap-input100 validate-input m-b-12">
+              <input id="userPassword" class="input100" type="password" name="pass">
+            </div>
+
+
+
+            <div class="container-login100-form-btn">
+              <button id="logForm" class="login100-form-btn">
+                Login
+              </button>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="drop" id="dropDownSelect1"></div>
+
+  </div>
+
+
+  <div class="main" id="mainPage">
+    <h1 class="user" id="userName"></h1>
+    <div class="form" id="filterForm">
+      <p class="filter">Filter</p>
+      <label>Name:</label><br>
```

Comment:
Correct HTML for label and input:
```
<label for="firstName">First Name:</label>
<input id="firstName"
       type="text"
       placeholder="First name">
```
*for* attribute value in `label` = *id* attribute value in `input`